Jeffkungu/WeatherForecast
# Create a Weather Forecast App

## What does this PR do?
#### Creates an android app that allows user to view weather conditions for various locations.
## Description of Task to be completed?
#### Users are able to search for various locations, view the weather conditions of a specific location  save the wether information locally on the device and delete the saved information.
## How should this be manually tested?

- #### After cloning the project, run it using android studio on an android device.
- #### Search for location by editing in the editText with a search icon.
- #### Click the thermometer icon just below the searching editText, a small window should pop-up with weather information of the specific location. You can click save or cancel text. On-clicking the save text, information is stored on the device and to will be able to view it on the Bookmarked Locations page.
- #### Click the floating button at the bottom of the page. This should take you to a different page displaying all the bookmarked locations.
- #### On-clicking the Floating button with a delete icon, deletes all the saved locations.
